iroh-net/src/magic_endpoint.rs

```diff
@@ -15,12 +17,12 @@ use crate::{
     discovery::{Discovery, DiscoveryTask},
     dns::{default_resolver, DnsResolver},
     key::{PublicKey, SecretKey},
-    magicsock::{self, ConnectionTypeStream, MagicSock},
+    magicsock::{self, Handle},
     relay::{RelayMap, RelayMode, RelayUrl},
     tls, NodeId,
 };
 
-pub use super::magicsock::{ConnectionInfo, LocalEndpointsStream};
+pub use super::magicsock::*;
```

Comment:
I know it is a pain, but please avoid the glob reexport, so it is easy to see what is being reexported

Reply:
removed in cb151e2 and explicitly added in af56639